enable/trait_defs/ui/wx/rgba_color_editor.py

```diff
@@ -16,7 +16,7 @@
 import wx
 
 from enable.colors import color_table
-
+from enable.api import Label, Window
```

Comment:
This could result in import cycles waiting to happen. It may not happen by virtue of ordering (i.e. alphabetical) and importlib caching imported modules. I think we should just do `from enable.label import Label` and `from enable.window import Window` here.
(My understanding is that `api` module is for others using the library and for tests, as tests should mimic the environment used by the library user. The `api` module is not targeting enable itself.

Reply:
Good catch